tests/e2e/specs/blocks/content-helper/top-bar-icon.spec.ts

```diff
@@ -21,7 +21,7 @@ const pluginButton = 'button[aria-label="Parse.ly Editor Sidebar"]';
  * Tests for the PCH Editor Sidebar top bar icon.
  */
 describe( 'PCH Editor Sidebar top bar icon in the WordPress Post Editor', () => {
-	const contentHelperTitle = 'Performance DetailsRelated Top Posts';
+	const contentHelperTitle = 'Performance DetailsThis post is not published, so its details are unavailable.Related Top Posts';
```

Comment:
We didn't made any changes around this so any reason why we are changing this?

Reply:
When the icon is clicked, the side-panel opens and it performs API requests. Because this is done with an unpublished post, this produces an error, which would previously result in 4 API requests, and thus more delay. So under these conditions, the test would pass.

However, as latency is much smaller now (no retry), we're getting the error message faster, and thus the test fails if we do not include it.